### DISCUSSION TITLE
大佬，我这边目录字体是红色，不知道怎么办？能帮我看看嘛？

### DISCUSSION THREAD

**Discussion by Tlich** [sikouhjw/gdutthesis#127] [Q&A] [answered]
大佬，我这边目录字体是红色，因为盲审结果需要修改为黑色，大佬能帮我看一下嘛？

**Comment by sikouhjw** [comment, marked as answer]
`hyperlink=none`

**Reply by TsingWei** [reply]
好奇为啥模板默认是红色，我彩印胶装交上去后学院也没说有问题（不排除他们没认真看）。上万方随便找了几篇看起来是latex写的，他们超链接颜色都改成黑色了，但我个人觉得还是有颜色比较好，方便阅读。不清楚如果最后提交电子版还是红色的话会不会不给过。

**Reply by sikouhjw** [reply]
估计是没仔细看，一般来说印刷版不需要超链接，电子版才需要，建议还是把超链接的颜色去掉
